src/main/java/com/liefery/android/stop_badge/StopBadgeView.java

```diff
@@ -180,7 +180,7 @@ protected void onLayout(
         if ( cache == null ) {
             int width = right - left;
             int height = bottom - top;
-            int size = Math.min( width, height );
+            int size = Math.max( width, height );
```

Comment:
I'm convinced that this change is wrong. If width and height differ in size, and we chose the larger dimension, the circle radius will exceed the smaller dimension.